lib/bundle/core+track.js

```diff
@@ -4,7 +4,12 @@ import { Track } from '../track';
  /**
   * @param {object} [cfg]
   */
-export default function Ravelin(cfg) {
+function Ravelin(cfg) {
+  cfg.Promise = Ravelin.Promise;
```

Comment:
Does this need to be:

```suggestion
  cfg.Promise = cfg.Promise || Ravelin.Promise;
```